threads/views.py

```
@@ -72,23 +71,64 @@ def get_reactions_count(self, request, *args, **kwargs):
+    @action(detail=True, methods=["get"],url_path="reactions-count")
+    def get_reactions_count(self, request, *args, **kwargs):
+       reaction_counts = (
+            CommentReactions.objects
+            .filter(comment=self.get_object())
+            .values('reaction')
+            .annotate(count=Count('id'))
+        )
+       return Response(reaction_counts)
+
+
+class ReplyViewSet(viewsets.ModelViewSet):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)